### PR TITLE
Signup tweaks

### PR DESCRIPTION
**Pull request by corsacca**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -65,6 +68,38 @@ function dt_pcsu_signup_blog_notification_email_rename( $message, $domain, $path
         What is your target location or people group? <span class="private-notice">Answer is kept private.</span>
     </label>
     <input type="text" id="dt_reason_for_subsite" name="dt_reason_for_subsite">
+    <p>
+        <label>Chose a Campaign Type:</label>
+            <?php
+            // @todo enable this
+            // $wizard_types = apply_filters( 'dt_campaigns_wizard_types', [] );
```
Comment (squigglybob) [inline functions.php]: Yeah I was just wondering about this too.
It would be nice if this was autoupdating from the filter

File: functions.php [hunk 2]
```diff
@@ -49,24 +49,67 @@ function dt_pcsu_signup_blog_notification_email_rename( $message, $domain, $path
...
+    </p>
+
+
+    <p>
+        <label for="dt_newsletter">
+            <input id="dt_newsletter" type="checkbox" name="dt_newsletter" checked>
+            <strong>Sign me up for Pray4Movement news.</strong>
+        </label>
+        <label for="p4m_agreement">
+            <input id="p4m_agreement" type="checkbox" name="p4m_agreement" checked required>
```
Comment (squigglybob) [inline functions.php]: should it start unchecked, like most other websites do, so that they have to actively read and click on the agreement?

**Review by squigglybob — COMMENTED** [review]
Nice work. Looks good.

When would we be planning to get that @todo point about getting the list of porches from the filter?

File: functions.php (hunk 2 above)
Comment (corsacca) [reply functions.php]: Ah, good point.
We don't have anything else right now for them to read but that line.

File: functions.php (hunk 1 above)
Comment (corsacca) [reply functions.php]: At some point :D

Comment (corsacca) [reply functions.php]: @squigglybob I added this as a new issue to the backlog.

**Pull request merged**